components/UI/textField.tsx

```diff
@@ -51,6 +53,7 @@ const TextField: FunctionComponent<TextFieldProps> = ({
           InputProps={{
             classes: {
               root: styles.textfield,
+              input: variant === "white" ? styles.inputWhite : "",
             },
```

Comment:
I'm not sure but if we can add that as css property it's better.

Reply:
you mean using a classname ? instead of InputProps and then classes ?